groups/application/profiles/heritage-live-eu-west-2/vars

```diff
@@ -28,14 +28,6 @@ weblogic_client_ips = [
 
 ####  Commented out until ready to include  ####
 #
-## Frontend ASG settings
-#gfn_app_instance_size = "t3.medium"
-#gfn_app_min_size = 1
-#gfn_app_max_size = 1
-#gfn_app_desired_capacity = 1
-#gfn_app_scaling_schedule_stop = "00 20 * * 1-5"
-#gfn_app_scaling_schedule_start = "00 06 * * 1-5"
-#
 #gfn_app_cw_logs = {
```

Comment:
You can remove logs as well, because the VM is lift/shift it wont have the correct setup that we use for our custom AMIs. Any work for the CloudWatch agent will need to be manual i.e. installed/setup/managed.